.cirrus.star

```diff
@@ -1,4 +1,40 @@
+# Check Starlark specs: https://github.com/bazelbuild/starlark/blob/master/spec.md
+# RE
 load("github.com/SonarSource/cirrus-modules@v2", "load_features")
+# Modules
+load(".cirrus/modules/env.star", "env")
+load(
+    ".cirrus/modules/build.star",
+    "build_task",
+    "build_test_analyze_task",
+    "sca_scan_task"
+)
+load(
+    ".cirrus/modules/qa.star",
+    "qa_os_win_task",
+    "qa_plugin_task",
+    "qa_ruling_task"
+)
+load(".cirrus/modules/promote.star", "promote_task")
+
+
+def merge_dict(target, source):
+    for key in source.keys():
+        if target.get(key) == None:
+            target.update({key: source[key]})
+        else:
+            target[key].update(source[key])
+
 
 def main(ctx):
-    return load_features(ctx)
+    conf = dict()
+    merge_dict(conf, load_features(ctx))
+    merge_dict(conf, env())
+    merge_dict(conf, build_task())
+    merge_dict(conf, build_test_analyze_task())
+    merge_dict(conf, qa_os_win_task())
+    merge_dict(conf, sca_scan_task())
+    merge_dict(conf, qa_plugin_task())
+    merge_dict(conf, qa_ruling_task())
+    merge_dict(conf, promote_task())
```

Comment:
In general, the structure is clear to me. However, I would expect to provide the project/repository-related values as arguments of the function call.

Reply:
I don't understand what you mean.
Can you provide an example, please?